packages/bpk-component-popover/stories.js

```diff
@@ -36,44 +38,65 @@ const Paragraph = withDefaultProps(BpkText, {
   className: getClassName('bpk-popover-paragraph'),
 });
 
-class PopoverContainer extends Component {
+type Props = {
+  ...$Diff<PopoverProps, { children: Node }>,
+  id: string,
+  changeProps: boolean,
+  targetFunction: ?Function,
```

Comment:
Another place to refine this type

Reply:
Ended up with `targetFunction: ?() => ?HTMLElement,`